### ISSUE TITLE
`GetSnapshotsAsync`: Unhandled exception. Newtonsoft.Json.JsonSerializationException: Required property 'latestTrade' expects a value but got null

### ISSUE THREAD

**Issue by gjtorikian** [alpacahq/alpaca-trade-api-csharp#358]
Given the following code:

```csharp
List<string> arr = new();
arr.Add("DILA");
await alpacaDataClient.GetSnapshotsAsync(arr).ConfigureAwait(false);
```

It crashes:

```
Unhandled exception. Newtonsoft.Json.JsonSerializationException: Required property 'latestTrade' expects a value but got null. Path 'DILA', line 1, position 202.
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.EndProcessProperty(Object newObject, JsonReader reader, JsonObjectContract contract, Int32 initialDepth, JsonProperty property, PropertyPresence presence, Boolean setDefaultValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateObject(Object newObject, JsonReader reader, JsonObjectContract contract, JsonProperty member, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.PopulateDictionary(IDictionary dictionary, JsonReader reader, JsonDictionaryContract contract, JsonProperty containerProperty, String id)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateObject(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.CreateValueInternal(JsonReader reader, Type objectType, JsonContract contract, JsonProperty member, JsonContainerContract containerContract, JsonProperty containerMember, Object existingValue)
   at Newtonsoft.Json.Serialization.JsonSerializerInternalReader.Deserialize(JsonReader reader, Type objectType, Boolean checkAdditionalContent)
   at Newtonsoft.Json.JsonSerializer.DeserializeInternal(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonSerializer.Deserialize(JsonReader reader, Type objectType)
   at Newtonsoft.Json.JsonSerializer.Deserialize[T](JsonReader reader)
   at Alpaca.Markets.HttpResponseMethodExtensions.DeserializeAsync[TApi,TJson](HttpResponseMessage response)
   at Alpaca.Markets.HttpResponseMethodExtensions.DeserializeAsync[TApi,TJson](HttpResponseMessage response)
   at Alpaca.Markets.HttpClientExtensions.callAndDeserializeAsync[TApi,TJson](HttpMessageInvoker httpClient, HttpRequestMessage request, CancellationToken cancellationToken)
   at Alpaca.Markets.HttpClientExtensions.callAndDeserializeAsync[TApi,TJson](HttpMessageInvoker httpClient, HttpMethod method, Uri endpointUri, CancellationToken cancellationToken)
   at Alpaca.Markets.HttpClientExtensions.GetAsync[TKeyApi,TValueApi,TKeyJson,TValueJson](HttpClient httpClient, UriBuilder uriBuilder, IEqualityComparer`1 comparer, Func`2 elementSelector, CancellationToken cancellationToken)
   at Alpaca.Markets.AlpacaDataClient.GetSnapshotsAsync(IEnumerable`1 symbols, CancellationToken cancellationToken)
```

Fairly easy to verify with curl, too:

```
curl -X GET -H "APCA-API-KEY-ID: xxx" -H "APCA-API-SECRET-KEY: xxx" "https://data.alpaca.markets/v2/stocks/snapshots?symbols=DILA"
{"DILA":{"latestTrade":null,"latestQuote":{"t":"2021-08-04T13:16:23.900405512Z","ax":"Q","ap":10.15,"as":5,"bx":"Q","bp":5,"bs":1,"c":["R"],"z":"C"},"minuteBar":null,"dailyBar":null,"prevDailyBar":null}}
```

**Comment by OlegRa** [comment]
@gjtorikian Please, test and close it if the problem is fixed now. Fix applied for all supported versions: 4.0.x, 5.0.x, and 5.1.x branches.

**Comment by gjtorikian** [comment]
Fix confirmed. Thanks!
